pkg/util/util.go

```diff
@@ -39,3 +40,28 @@ func ValidateDriverName(driverName string) error {
 	}
 	return err
 }
+
+func CleanTargetPath(targetPath string) {
+	// remove the targetPath parent directory
+	// resolve kubelet issue https://github.com/kubernetes/kubernetes/issues/60987
```

Comment:
No need to mention the issue here. It should be explained in the commit message what the issue is, **and** add a link to the issue there.